lib/nvmf/rdma.c

```diff
@@ -826,6 +826,18 @@ spdk_nvmf_request_prep_data(struct spdk_nvmf_request *req)
 		req->xfer = spdk_nvme_opc_get_data_transfer(req->cmd->nvmf_cmd.fctype);
 	} else {
 		req->xfer = spdk_nvme_opc_get_data_transfer(cmd->opc);
+		if ((cmd->opc == SPDK_NVME_OPC_GET_FEATURES) ||
```

Comment:
This needs an extra check to verify that this is an admin queue; the admin command set opcodes overlap with the NVM command set opcodes.